### ISSUE TITLE
[Request] Add ability to choose different quota timeout (for paid accounts)

### ISSUE THREAD

**Issue by ryanjoachim** [ericyd/gdrive-copy#72]
### Are you requesting a feature or reporting a bug?
**Feature**
### What is the current behavior?
Script hits 90 minute quota max (default for free/consumer account)
### What is the expected behavior?
It would be great to be able to either auto-detect the account type (if even possible), or have the option to choose "I have a business G Suite account) to be able to make use of the higher quota time available for paid accounts.
### Specify your

* operating system:
* browser:

**Comment by marcussacana** [comment]
Since this feature looks will release Soon™
I did the changes by my self as said in the #120
I don't know why no one did this before yet but,
If you don't want lost time creating your own fork, here is a link to my fork:
https://script.google.com/macros/s/AKfycby0C5GU_zclQDkAfVT3G-S0x7_AoucWpZgJuSY1/exec
and the source:
https://script.google.com/d/1aNwhWTSHm5JsW8FUEWX3RoPZek8HiBmQThDqFf2LzAcz6eRFEIwxhVn-/edit?usp=sharing

**Comment by ericyd** [comment]
That is very generous to assume this will be done soon :grin:
I am always open to PRs if you'd like to contribute to the main branch!

**Comment by marcussacana** [comment]
I don't think you will approve my code style, I don't respect any pattern of code. just give a look in the hell that is my github profile

**Comment by paritel99** [comment]
> Since this feature looks will release Soon™

I did the changes by my self as said in the #120
I don't know why no one did this before yet but,
If you don't want lost time creating your own fork, here is a link to my fork:
https://script.google.com/macros/s/AKfycby97tHESvVu68bRhIHiz5_t0jbYpyyP1wLsc99wJVxjW1nuhqc/exec
and the source:
https://script.google.com/d/1aNwhWTSHm5JsW8FUEWX3RoPZek8HiBmQThDqFf2LzAcz6eRFEIwxhVn-/edit?usp=sharing

It is a very nice script it's work. Thank you

**Comment by Litardo33** [comment]
> Since this feature looks will release Soon™
> I did the changes by my self as said in the #120
> I don't know why no one did this before yet but,
> If you don't want lost time creating your own fork, here is a link to my fork:
> https://script.google.com/macros/s/AKfycby97tHESvVu68bRhIHiz5_t0jbYpyyP1wLsc99wJVxjW1nuhqc/exec
> and the source:
> https://script.google.com/d/1aNwhWTSHm5JsW8FUEWX3RoPZek8HiBmQThDqFf2LzAcz6eRFEIwxhVn-/edit?usp=sharing

This script Stopped in 6 hours. Is there a solution to dont stop?

**Comment by marcussacana** [comment]
Ask the google :)
I don't know, Wrote that to use with my student g-suite account, if your account have bigger limitations you will need like me, download and change the source.

**Comment by Ltd333** [comment]
> Ask the google :)
> I don't know, Wrote that to use with my student g-suite account, if your account have bigger limitations you will need like me, download and change the source.

OK! I also use a student G-suite account. Thank you. Good Luck!

**Comment by wynstep** [comment]
> Since this feature looks will release Soon™
> I did the changes by my self as said in the #120
> I don't know why no one did this before yet but,
> If you don't want lost time creating your own fork, here is a link to my fork:
> https://script.google.com/macros/s/AKfycby97tHESvVu68bRhIHiz5_t0jbYpyyP1wLsc99wJVxjW1nuhqc/exec
> and the source:
> https://script.google.com/d/1aNwhWTSHm5JsW8FUEWX3RoPZek8HiBmQThDqFf2LzAcz6eRFEIwxhVn-/edit?usp=sharing

Resuming the issue, cannot sign in to forked app script. Could you please help?

**Comment by marcussacana** [comment]
It's because my g-suite account has revoked, try use the second link to fork and run yourself

**Comment by wynstep** [comment]
> It's because my g-suite account has revoked, try use the second link to fork and run yourself

Thanks for your reply. I run the fork on my account but I continuously receive an error when I try to run the web app, any suggestion please? Newbie here :-)

**Comment by wynstep** [comment]
> > It's because my g-suite account has revoked, try use the second link to fork and run yourself
> 
> Thanks for your reply. I run the fork on my account but I continuously receive an error when I try to run the web app, any suggestion please? Newbie here :-)

Just a short update, the web app (from fork) works on my personal account (gmail.com) but does not work from gsuite for education account. Any suggestion please?

**Comment by marcussacana** [comment]
Not sure, my gsuite is really suspended since I didn't paid again, but if works in the personal account looks like the script still alive.
